CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,5 @@
+- Add: openmetrics-compatible `/metrics` endpoint in nortbound API (#1627)
+- Remove: push-based stats
```

Comment:
```suggestion
- Remove: push-based stats (including stats section in config file)
```

(If I'm right in https://github.com/telefonicaid/iotagent-node-lib/pull/1629#pullrequestreview-2218265090)